### ISSUE TITLE
Quickfix list window related problems

### ISSUE THREAD

**Issue by wheatdog** [hecal3/vim-leader-guide#7]
If quickfix list window is opened, the number of empty buffer will increase every time I open vim-leader-guide buffer.

![output](https://cloud.githubusercontent.com/assets/7741908/14133381/2cdb8580-f614-11e5-82c3-b4cccc5c9fc8.gif)

If I vertical split my window and open the quickfix list window, the height of quickfix list window will grow every time I open vim-leader-guide buffer.

![output](https://cloud.githubusercontent.com/assets/7741908/14133466/047870fc-f615-11e5-84cf-d08d0673deab.gif)

**Comment by hecal3** [comment]
The second issue should be fixed now.

I will have to look into the first one later.

**Comment by hecal3** [comment]
The first issue is 'fixed' as well... somewhat.

Apparently running `split` or `vsplit` in the quickfix- or locationlist window will always create a new buffer. I found no way to avoid that.

My only solution is to wipe the extra buffers immediately after creation. It only happens when pressing `leader` while the cursor is in the location list window, so it really shouldn't occur too often.

Not exactly an elegant solution, but the best I came up with.
